Svc/PolyDb/test/ut/Readme.txt

```diff
@@ -1,9 +1,9 @@
 This test can be run by executing the following:
```

Comment:
Just delete this file?  We no longer maintain per-component readmes.

Reply:
Removed.